Dyno-RPG (#74)
DynoCC/Dyno-Custom-Commands: Fun Commands/Mini-Games/Dyno-RPG/register.txt

{delete}
{silent}
{!role +Adventurer}

> {!announce {#the-world-gates} Welcome to the world of **Avelon**, {user.username}! This is a cruel world that the gods have long since forsaken. Darkness and torment lies around every corner, and I can not guarantee that you will maintain your sanity. But rest assured, the **Divine Council** will watch over you. Now, head forth with caution young Adventurer, and claim your place in this forsaken world.}

Reviewer: Small possible spelling change
```suggestion
{!announce {#the-world-gates} Welcome to the world of **Avelon**, {user.username}! This is a cruel world that the gods have long since forsaken. Darkness and torment lies around every corner, and I cannot guarantee that you will maintain your sanity. But rest assured, the **Divine Council** will watch over you. Now, head forth with caution young Adventurer, and claim your place in this forsaken world.}
```

Author: Fixed

> {!announce {#the-crossroads} {user.username} has arrived {#the-wrold-gates}.}

Reviewer: Through line 11, world is spelled incorrectly.

Author: Fixed
